examples/nlp_seq/train.py

```diff
@@ -191,35 +192,38 @@ def compute_metrics(logits, labels, weights):
   return metrics
 
 
-def train_step(optimizer, batch, learning_rate_fn, model, dropout_rng=None):
+def train_step(state,
+               batch,
+               model,
+               learning_rate_fn,
+               dropout_rng=None):
   """Perform a single training step."""
-
   train_keys = ['inputs', 'targets']
   (inputs, targets) = [batch.get(k, None) for k in train_keys]
 
-  weights = jnp.where(targets > 0, 1, 0).astype(jnp.float32)
-  dropout_rng, new_dropout_rng = random.split(dropout_rng)
+  weights = jnp.where(inputs > 0, 1, 0).astype(jnp.float32)
```

Comment:
Nit: is this change from `targets` to `inputs` intended?

Reply:
Very well spotted. No, this was not intended - makes not difference as both are padding. However, let's move back to targets. I will update.

Reply:
Thanks!